doc/HOWTO_oracle.md

```diff
@@ -4,145 +4,145 @@ Follow these instructions to set up the Oracle database to run in Docker, but th
+    ```
+    brew install instantclient-basic
+    brew install instantclient-sqlplus
+    brew install instantclient-sdk
+    ```
 
-##### Add Environment Variables to profile
+### Set Up Environment Variables
 
-* Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
+1. Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
 
 ```
-export NLS_LANG="AMERICAN_AMERICA.AL32UTF8"
-export ORACLE_HOME=/opt/oracle/instantclient
-export OCI_DIR=$ORACLE_HOME
+# The Oracle adapter for ActiveRecord uses this password to connect as a
+# system user, to be able to create and drop databases appropriately.
+export ORACLE_SYSTEM_PASSWORD=Oradoc_db1
+
+# This is used to specify the default language and encoding for Oracle clients
+export NLS_LANG="AMERICAN_AMERICA.UTF8"
 ```
 
-* `source ~/.profile` or `source ~/.zprofile`, respectively
+1. `source ~/.profile` or `source ~/.zprofile`, to load the changes you just made
 
 
-##### Test your installation:
+### Test Your Installation
 
-* Run this command:
+To connect to the Oracle server, run:
 
 ```
-sqlplus myuser/mypassword@//myserver:1521/mydatabase.mydomain.com
+sqlplus "sys/Oradoc_db1@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))" as sysdba
 ```
 
-* Verify you see an output like this:
+Your output should show something like this:
 
 ```
-SQL*Plus: Release 12.1.0.2.0 Production on Fri Mar 3 13:48:51 2017
+Connected to:
+Oracle Database 12c Enterprise Edition Release 12.2.0.1.0 - 64bit Production
 
-Copyright (c) 1982, 2016, Oracle.  All rights reserved.
-
-ERROR:
-ORA-12154: TNS:could not resolve the connect identifier specified
-
-
-Enter user-name:
+SQL> 
 ```
 
-* `^D` out of that and carry on.
-
 ## Setting up the Database
 
-##### Install Required Gems
-
-* Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems
-enabled.
-
-##### Set up database configuration
-
-* Copy from oracle settings template
-
-```
-cp config/database.yml.oracle.template config/database.yml
-```
-
-* Validate keys in settings are correct for your current application
-
-* Start, setup, and seed the oracle container:
-
-```
-docker/oracle/setup.sh
-# demo:seed is optional
-rake demo:seed
-```
+1. Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems enabled in your `Gemfile`.
+1. Copy the Oracle template for `database.yml` by running `cp config/database.yml.oracle.template config/database.yml`.
+1. Run `bundle exec rake db:setup`.
+1. Optionally, run `bundle exec rake demo:seed` if you want to populate the database with demo data.
 
 # Optional Extras
 
-##### Install Oracle SQL Developer
+## Install Oracle SQL Developer
 
 * Download from: `http://www.oracle.com/technetwork/developer-tools/sql-developer/overview/index.html`
 
 * Install into `/Applications`
 
-#### Restore from a backup
+## Restore From Backup
 
 Run `bundle exec rake db:oracle_drop_severe`. This will ensure that your database
```

Comment:
We'll probably address this separately, but when loading from a dmp, while I haven't gotten it successfully running, I do believe that `db:reset` or `db:setup` won't work because when you try to load the file, it will complain about the tables already existing. So I think these instruction should probably include `db:drop db:create`. And hopefully get rid of `db:oracle_drop_severe`.

Reply:
Good call! I just did this in 98402df and subsequently removed the now-unneeded `db:oracle_drop*` tasks—the regular `oracle_drop` isn’t needed since `db:drop` now works properly.